test/index.js

```diff
@@ -71,14 +71,14 @@ describe('logging-winston', function() {
     fakeLogInstance = {};
     fakeLoggingOptions_ = null;
     fakeLogName_ = null;
-    loggingWinston = new LoggingWinston(OPTIONS);
+    loggingWinston = new LoggingWinston.LoggingWinston(OPTIONS);
```

Comment:
nit: Do the `.LoggingWinston` at the `require` rather than distributing it in the code.

Reply:
Since we have a test to cover initializing an object without new, we need to use this signature to avoid undefined this.